### PR TITLE
update communications links in readme

### PR DESCRIPTION
**Pull request by kirstenalarsen**
#### What? Why?

Communication / join the community links were out of date and obsolete. Now includes links to the community forum and self-invite to Slack

#### What should we test?

nothing

#### Release notes

Changed

#### How is this related to the Spree upgrade?

Not

### CODE REVIEW

**Comment by kirstenalarsen** [comment]
i just updated the file with your change, in commit on same branch. guess . . probably wrong

File: README.md [hunk 1]
```diff
@@ -6,12 +6,15 @@
...
-Supported by the Open Food Foundation, we are proudly open source and not-for-profit - we're trying to seriously disrupt the concentration of power in global agri-food systems, and we need as many smart people working together on this as possible.
+Supported by the Open Food Foundation and a network of global affiliates, we are proudly open source and not-for-profit - we're trying to seriously disrupt the concentration of power in global agri-food systems, and we need as many smart people working together on this as possible.
 
 We're part of global movement - get involved!
 
 * Fill in this short survey to tell us who you are and what you want to do with OFN: https://docs.google.com/a/eaterprises.com.au/forms/d/1zxR5vSiU9CigJ9cEaC8-eJLgYid8CR8er7PPH9Mc-30/edit#
-* Find out more and join in the conversation - http://openfoodnetwork.org
+* Find out more and join in the conversation:
+  * Join our [Slack channel](https://openfoodnetwork.org/slack-invite). Introductions on #general are welcome! 
+  * Join our discussion forum - https://community.openfoodnetwork.org
```
Comment (sauloperez) [inline README.md]: we can have `discussion forum` as link as well :ok_hand:

**Comment by mkllnk** [comment]
That was very good @kirstenalarsen. You can do the same with the discussion forum (same URL to communit..., but link the words `discussion forum` to there.

**Comment by sauloperez** [comment]
I went ahead and added the link so that we can close this soon. I hope you don't mind @kirstenalarsen I understand you don't have too much time to spend on PRs.

**Comment by mkllnk** [comment]
Oh dear. Rob's changes got merged first. He changed a lot of text and included some of these links already. @kirstenalarsen It's probably best if you read the current version and check if you still want to change something, then open a new pull request if needed.

**Pull request closed without merging**